api/sspanel/sspanel.go

```diff
@@ -804,10 +782,7 @@ func (c *APIClient) ParseSSPanelNodeInfo(nodeInfoResponse *NodeInfoResponse) (*a
 		tlsType = "tls"
 		transportProtocol = "tcp"
 
-		// Select security type
-		if nodeConfig.EnableXtls == "1" {
```

Comment:
The param `EnableXtls` of custom config have been removed?

Reply:
By the look of the origin code, this config is never properly implemented, in Clash.Meta config, xtls feature is enabled via `flow` instead of `security`, and `security` is a Vmess-only param, so the whole block of code should all be removed.

Reply:
And do you plan to add REALITY support? If not then the whole xtls and vless related code can all be removed since they are no longer maintained and marked outdated in the upstream.

Reply:
REALITY have already been supported. But only use local config now.

Reply:
Well that's just confusing for the operator, isn't it? The custom config should have all the necessary parameters already when Clash.Meta support is added.

Reply:
Can you submit a PR for support REALITY config from web api?

Reply:
Again I don't have a plan to add any new node type to the XrayR since it's the actual repo maintainer's job, but I  recommend you use a dedicated node type for anything related to Xray's "genius inventions" like what v2board did with their VLESS node type instead of mashing all the possible configuration together. 

The reason SSPanel doesn't have that is because we never officially adapt Vless or REALITY and current support is "experimental" at best, if people started to move toward those new quic-based solutions(tuic etc.) then it's highly likely to stay that way.

Reply:
https://github.com/sspanel-uim/SSPanel-Uim-Dev/blob/6b512cda7c844dd559a668364713516fbe8934e1/src/Services/Subscribe/Clash.php#L101-L148C24

All the param names needed have been included in this code block. I can't do anything with the InboundBuilder without major refactoring in place, otherwise, I would just be adding mess to the already complex spaghetti.

This PR is the best I can do for now until I have more time to read through the rest of codebase.

Reply:
Could you give me a complete json data for custom config? So that I can build a mock server for deal with it.

Reply:
Should look like this https://github.com/XrayR-project/XrayR/issues/452, don't know why he/she closed the issue though.